### DISCUSSION TITLE
Why I can't change map orientation ?

### DISCUSSION THREAD

**Discussion by jclaudej** [devemux86/cruiser#115] [Q&A] [answered]
HI,

When I want to setup my direction of travel: (Parameter/Navigation) it's not possible to modify it ?
And top of that  => direction of travel,  is grayed out ?
And my map is always oriented to the north ? While the parameter of direction of travel is (Direction of 3D movement) ?
I have the last version of Cruiser 3.0.15.
Perhaps it will be interresting to manage this fonctionnality more easely ;)

**Comment by devemux86** [comment, marked as answer]
Hi @jclaudej,

Did you change the "Settings | Map | Advanced | Rendering engine"?
(or search settings for "engine")

The default setting is "VTM (OpenGL)" which supports map rotation.

**Reply by jclaudej** [reply]
Ok, thank a lot for your reactivity ;)
My setup "Rendering engine" was Mapsforge.

I test soon as possible ;)
